index.bs

```diff
@@ -4033,7 +4038,7 @@ that have [=members=] with these names.
 
         typeof SessionManager.prototype.values;            // Evaluates to "function"
         var it = sm.values();                              // values() returns an iterator object
-<!--    String(it);                                        // Evaluates to "[object SessionManagerIterator]"
+<!--    String(it);                                        // Evaluates to "[object SessionManager Iterator]"
                                                            // TODO: https://github.com/heycam/webidl/issues/419 -->
```

Comment:
This should’ve been uncommented in this PR:
```suggestion
        String(it);                                        // Evaluates to "[object SessionManager Iterator]"
````

I’m fixing this in <https://github.com/heycam/webidl/pull/858>.